src/constants/vocabulary.ts

```diff
@@ -7,6 +7,7 @@ export const SCRIPT_DTO = `${S_PIPES}/script-dto`;
 export const QUESTION_DTO = `${S_PIPES}/question-dto`;
 export const ROOT_QUESTION = `${S_PIPES}/has-root-question`;
 export const SCRIPT_PATH = `${S_PIPES}/has-script-path`;
+export const ORIGINAL_SCRIPT_PATH = `${S_PIPES}/has-original-script-path`;
```

Comment:
```suggestion
// relates a module to a path of script from which the module was executed.
export const EXECUTION_SCRIPT_PATH = `${S_PIPES}/has-execution-script-path`;
```

Unfortunatelly, I cannot imagine anything when you say has-original-script-path ... i suggest to rename it to has-execution-script-path, i.e. path from which script is executed.